.bit/responses/1.3-Week 1 Step 3.md

```diff
@@ -55,31 +56,17 @@ A "RESTful" API is an API that follows a few different rules/coding practices wh
 
 Clients communicate with servers with through HTTP requests. For example, when you are on your favorite browser and look up YouTube.com, you are making an HTTP "get" request to the server to load the contents from YouTube.com.
 
-<details>
-<summary>Get Request</summary>
-A "Get Request" gets data from a server (hence the name). The data we want is specified using a URL we call a Request URL. In this case, you will use a Get Request URL from the Catass API to receive a cat picture.
-
+#### Common Request Types
+Below are some of the most common HTTP requests. Read through each one, and try to get familiar with their functions!
```

Comment:
I think all of these can be placed in a drop down since some people may already be familiar with them. (the different requests)